f5_openstack_agent/lbaasv2/drivers/bigip/service_adapter.py

```diff
@@ -482,7 +483,12 @@ def _map_virtual(self, loadbalancer, listener,
         self._add_vlan_and_snat(listener, vip)
         self._add_profiles_session_persistence(listener, pool, vip)
 
-        # Set bandwidth contoller
+        # Set bandwidth controller from loadbalancer's setting.
+        # applying order:
+        #   loadbalancer's bandwidth field -> l7policy's name(bwc name)
```

Comment:
it seems we have two methods to configure the same BWC function:
  1. use loadbalancer setting (seems it can only be low mid high) 
  2. use l7policy setting (seems it can be set a random value)

Reply:
Right.